projects/rocr-runtime/runtime/hsa-runtime/image/image_manager_gfx12.cpp

```diff
@@ -664,54 +669,52 @@ uint32_t ImageManagerGfx12::GetAddrlibSurfaceInfoNv(
   in.width = width;
   in.height = height;
   in.numSlices = num_slice;
-  in.pitchInElement = image_data_row_pitch / image_prop.element_size;
```

Comment:
Do we still need  in.pitchInElement = image_data_row_pitch / image_prop.element_size;?